demo/demo.html

```diff
@@ -3,20 +3,24 @@
   <head>
     <title>Demo of Logic and View Integration</title>
     <script src="../dist/components.js"></script>
-    <script src="https://cdn.groupbycloud.com/sfx-all-plugins-bundle.js"></script>
+    <script src='./temp/sfx-all-plugins-bundle.js'></script>
+    <!-- above tag to be removed when the cdn PR passes with the updated bundle -->
+    <!-- <script src="https://cdn.groupbycloud.com/sfx-all-plugins-bundle.js"></script> -->
   </head>
   <body>
-    <sfx-search-box placeholder="search" clearbutton searchbutton></sfx-search-box>
-    <sfx-sayt visible></sfx-sayt>
+    <sfx-search-box id="searchbox1" placeholder="search" clearbutton searchbutton></sfx-search-box>
+    <sfx-sayt searchbox="searchbox1"></sfx-sayt>
+    <sfx-search-box id="searchbox2" placeholder="search" clearbutton searchbutton></sfx-search-box>
+    <sfx-sayt searchbox="searchbox2" minSearchLength=5></sfx-sayt>
     <sfx-list class="product-grid"></sfx-list>
 
     <script>
       // init core and plugins
       const core = new sfxCore.Core();
       const domEventsPlugin = new sfxPlugins.DomEventsPlugin()
-      const saytPlugin = new sfxPlugins.SaytPlugin({subdomain: 'apparel', collection: 'products'});
+      const saytPlugin = new sfxPlugins.SaytPlugin({subdomain: "apparel", collection: "products"});
```

Comment:
Please use spaces inside of `{...}` in JS. 
```suggestion
      const saytPlugin = new sfxPlugins.SaytPlugin({ subdomain: "apparel", collection: "products" });
```